README.md

```diff
@@ -28,20 +45,761 @@ data portals
 - 🚀 NextJS framework: so everything in NextJS for free React, SSR, static site generation, huge number of examples and integrations etc.
   - SSR => unlimited number of pages, SEO etc whilst still using React.
   - Static Site Generation (SSG) (good for small sites) => ultra-simple deployment, great performance and lighthouse scores etc
-- 📋 Typescript support
 
-### Check out more of the examples
+# Installation and setup
+Before installation, ensure your system satisfies the following requirements:
+
+- Node.js 10.13 or later
+- Nextjs 10.0.3
+- MacOS, Windows (including WSL), and Linux are supported
+
+> Note: We also recommend instead of npm using `yarn` instead of `npm`.
+> 
+Portal.js is built with React on top of Nextjs framework, so for a quick setup, you can bootstrap a  Nextjs app and install portal.js as demonstrated in the code below:
+
+```bash=
```

Comment:
From this point until Getting Started you sort of have your own mini getting started example. this is great and i wonder if we want to refactor this to flow a little clearer e.g. have this below.

That said, for *right now* i think this is fine and we should get this merged.